components/vf-back-to-top/vf-back-to-top.njk

```diff
@@ -0,0 +1,22 @@
+{# Make sure any variables are listed inside the following if statement #}
+{% if context %}
+  {% set type = context.type %}
+  {% set text = context.display_text %}
+  {% set scrollToId = context.scrollToId %}
+{% endif %}
+
+<div class="vf-back-to-top vf-back-top--{{type}}">
+  <a type="submit" data-scroll-to-id="{{scrollToId}}" class="vf-button vf-button--primary vf-button--sm" aria-label="{% if text %} {{text}} {% else %} Back to top {% endif %}">
+    <span class="vf-button__text | vf-u-sr-only">Back to top</span>
```

Comment:
I think you could drop the `vf-u-sr-only` and put your variables here, like:
```
    <span class="vf-button__text | vf-u-sr-only">
    {% if text %}
      {{text}}
    {% else %}
      Back to top
    {% endif %}
</span>
```

Reply:
not 100% sure if I understand, but I did this:
```
{% if context %}
  {% set text = context.text %}
 ...
{% endif %}

{# Determine text, if not explicitly set -#}
{% if not text %}
  {% set text = 'Back to top' %}
{% endif %}

<div class="vf-back-to-top vf-back-top--{{type}}" data-vf-js-back-to-top {% if type === 'floating'%} data-vf-js-back-to-top-floating {% endif %}>
  <a {% if scrollToId %} href="{{'#'+scrollToId}}" {% endif %}} data-scroll-to-id="{{scrollToId}}" class="vf-button vf-button--primary vf-button--sm" aria-label="{{text}}">
    <span class="vf-button__text | vf-u-sr-only">{{text}}</span>
     ...
      {{text}}
  </a>
```